webpack.web.js

```diff
@@ -32,6 +32,8 @@ module.exports = merge(common.webpackConfig, {
       template: "./src/web/index.html.ejs",
       filename: "index.html",
       chunks: ["index"],
+      ogImage: `${packageMeta.homepage}images/color-fb.jpg`,
+      twitterImage: `${packageMeta.homepage}images/color-twitter.jpg`,
```

Comment:
Does `packageMeta.homepage` get overridden per env, or will we need to update it to the prod URL before general release?

https://github.com/mozilla/FirefoxColor/blob/2852096f6e30627035c2ab852b74b719911d893b/package.json#L11

Reply:
This should probably use `process.env.SITE_URL` instead of packageMeta. That *does* get changed per environment

Reply:
okay cool